docs/integrations/unity-catalog-spark.md

```diff
@@ -0,0 +1,179 @@
+    (3, "air conditioner"),  
+    (4, "tea"),  
+], ["transaction_id", "item_name"])
+
+df.write.format("parquet").saveAsTable("stores.us_east.transactions")
+```
+
+This table uses the three-level Unity Catalog namespacing: `catalog_name.schema_name.table_name`.
+
+Confirm that we’re able to read the table:
+
+```
+spark.table("stores.us_east.transactions").show()
+
++--------------+---------------+  
+|transaction_id|      item_name|  
++--------------+---------------+  
+|             1|          socks|  
+|             2|          chips|  
+|             3|air conditioner|  
+|             4|            tea|  
++--------------+---------------+
+```
+
+Now, let’s create another managed table:
+
+```python
+voided = spark.createDataFrame([(1,), (4,)], ["transaction_id"])
+
+voided.write.format("parquet").saveAsTable("stores.us_east.voided")
+```
+
+Here’s a visualization of the tables.  
+
+![UC Spark 1](../assets/images/uc_spark1.png)
+
+Suppose the transactions table represents all the transactions, and the voided table contains the transactions that were subsequently canceled.  Compute all the non-voided transactions.
+
+```python
+transactions = spark.table("stores.us_east.transactions")  
+voided = spark.table("stores.us_east.voided")
+
+transactions.join(  
+    voided,   
+    transactions.transaction_id == voided.transaction_id,   
+    "leftanti"  
+).show()
+```
+
+```
++--------------+---------------+  
+|transaction_id|      item_name|  
++--------------+---------------+  
+|             2|          chips|  
+|             3|air conditioner|  
++--------------+---------------+
+```
+
+Now, let’s look at how to create external Unity Catalog tables.
+
+## Create an external Unity Catalog table with Spark
+
+Here’s how to create an external Unity Catalog table with Spark:
+
+```python
+letters = spark.createDataFrame([  
+    (1, "a"),   
+    (2, "b"),  
+    (3, "c"),  
+], ["id", "letter"])
+
+letters.write.format("parquet").saveAsTable("stores.whatever.letters", path="/tmp/letters")
+```
+
+When you manually specify the path, as in the example above, Spark creates an unmanaged table.
+
+Let’s look at a visualization of our current tables:
+
+![UC Spark 2](../assets/images/uc_spark2.png)
+
+See this page to learn more about the difference between external and managed Unity Catalog tables.
+
+## Spark write data to path
+
+You can also use Spark to write data to a path and not make any Unity Catalog entries.
+
+```python
+people = spark.createDataFrame([  
+    (1, "li"),   
+    (2, "chung"),  
+], ["id", "first_name"])
+
+people.write.format("parquet").save("/tmp/people")
+```
+
+As you can see in the example above, writing data with Spark doesn’t necessarily create a new entry in the Unity Catalog. This writes data to a path but doesn’t create any associated entries in the Unity Catalog, burdening the user with managing the credentials and access to the data themselves.
+
+## Advantage of Unity Catalog for Spark
```

Comment:
I like that there is a section like this, but these arguments for UC should be mentioned in the central docs of "why UC", not just in the integration doc of specific integrations. its perfectly okay for integration specific pages to be repeat that stuff, and only fine tune the arguments for each integration.